cli/update.go

```diff
@@ -53,4 +80,8 @@ func update() *cobra.Command {
 			return nil
 		},
 	}
+
+	cmd.Flags().BoolVar(&alwaysPrompt, "always-prompt", false, "Always prompt all parameters. Does not pull parameter values from existing workspace")
```

Comment:
`--reprompt` sounds better. Or we could keep it called `--always-prompt` and ask the user if they want to edit their parameters using a y/n prompt.

Reply:
Use `--always-prompt` and a `y/n` prompt? 

I am not sold on `always-prompt` either, was unsure on what exactly to use :thinking:

Reply:
I meant `--always-prompt`, but if not specified ask them if they want to change values with a y/n prompt

Reply:
I can add this prompt though if `always-prompt` is omitted :+1:

Reply:
Our `y/n` prompt only returns yes or error. We need to modify it to accept a proper `y/n` if we want to do something other than abort on `no`.